pyoverkiz/models.py

```diff
@@ -447,7 +447,7 @@ def __init__(
 
 @define(init=False, kw_only=True)
 class Scenario:
-    label: str
+    label: str = field(repr=obfuscate_string, default=None)
```

Comment:
What set a default value?

Reply:
I copied that, will remove it since str is required.